### PR TITLE
N-Queens Problem in c ,c++, JAVA and python

### PR DESCRIPTION
**Pull request by ShubhaSwarup**
Added code for solving N-Queens Problem in c ,c++, JAVA and python

#760 

Issue no #760 

## ✔️ Check List (Check all the applicable boxes) 

- [✅] My code follows the code style of this project.
- [✅] This PR does not contain plagiarized content.
- [✅] The title of my pull request is a short description of the requested changes.

## 📷 Screenshots
For c output:
Screenshot 2023-06-11 at 11 56 32 AM
For c++ output:
Screenshot 2023-06-11 at 11 59 46 AM
For java output:
image

For python output:
image

### CODE REVIEW

File: .vscode/tasks.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
```
Comment (Kumar-laxmi) [inline .vscode/tasks.json:1]: Kindly remove this file ```.vscode/tasks.json```

File: C++/Backtracking/N_Queen.dSYM/Contents/Info.plist [hunk 2]
```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Kumar-laxmi) [inline C++/Backtracking/N_Queen.dSYM/Contents/Info.plist:1]: Kindly remove this file ```C++/Backtracking/N_Queen.dSYM/Contents/Info.plist```

**Review by Kumar-laxmi — CHANGES_REQUESTED** [review]
Remove these files also: ```C++/Backtracking/N_Queen``` & ```C++/Backtracking/N_Queen.dSYM/Contents/Resources/DWARF/N_Queen```

File: C++/Backtracking/N_Queen.dSYM/Contents/Info.plist (hunk 2 above)
Comment (ShubhaSwarup) [reply C++/Backtracking/N_Queen.dSYM/Contents/Info.plist:1]: > Kindly remove this file `C++/Backtracking/N_Queen.dSYM/Contents/Info.plist`

okay i will remove these files

Comment (ShubhaSwarup) [reply C++/Backtracking/N_Queen.dSYM/Contents/Info.plist]: Removed. Please check again

**Review by zugzwang03 — APPROVED** [review]

**Review by Kumar-laxmi — APPROVED** [review]

**Review by Kumar-laxmi — CHANGES_REQUESTED** [review]
Please share the screen shot of your output and not the code

**Comment by ShubhaSwarup** [comment]
> Please share the screen shot of your output and not the code in the PR description

Done

**Review by Kumar-laxmi — CHANGES_REQUESTED** [review]
Place this program in ```Divide and Conquer```

**Comment by ShubhaSwarup** [comment]
> Place this program in `Divide and Conquer`

Done. Please check.

**Review by Kumar-laxmi — APPROVED** [review]

**Review by Kumar-laxmi — CHANGES_REQUESTED** [review]
Kindly put the screen shot of actual output in the PR description

**Review by 072arushi — APPROVED** [review]

**Review by Kumar-laxmi — CHANGES_REQUESTED** [review]
- The screen shots of Java & Python programs are missing
- You can give same name to all program files

**Comment by ShubhaSwarup** [comment]
> * The screen shots of Java & Python programs are missing
> * You can give same name to all program files

Done.Please check.

File: C++/Divide-and-Conquer/N_Queen
Comment (zugzwang03) [inline C++/Divide-and-Conquer/N_Queen:1]: remove this file

File: C/Divide-and-Conquer/N_Queen
Comment (zugzwang03) [inline C/Divide-and-Conquer/N_Queen:1]: remove this file

File: C/Divide-and-Conquer/N_Queen.dSYM/Contents/Resources/DWARF/N_Queen
Comment (zugzwang03) [inline C/Divide-and-Conquer/N_Queen.dSYM/Contents/Resources/DWARF/N_Queen:1]: remove this file

**Review by zugzwang03 — CHANGES_REQUESTED** [review]

File: C/Divide-and-Conquer/N_Queen.dSYM/Contents/Resources/DWARF/N_Queen
Comment (ShubhaSwarup) [reply C/Divide-and-Conquer/N_Queen.dSYM/Contents/Resources/DWARF/N_Queen:1]: > remove this file

done.please check

**Review by Kumar-laxmi — CHANGES_REQUESTED** [review]
Remove: ```.vscode/tasks.json```

**Review by zugzwang03 — APPROVED** [review]

**Pull request merged**